pkg/datapath/linux/config/config.go

```diff
@@ -718,11 +718,7 @@ func (h *HeaderfileWriter) writeTemplateConfig(fw *bufio.Writer, e datapath.Endp
 		fmt.Fprintf(fw, "#define FORCE_LOCAL_POLICY_EVAL_AT_SOURCE 1\n")
 	}
 
-	if e.RequireRouting() {
-		fmt.Fprintf(fw, "#define ENABLE_ROUTING 1\n")
-	}
-
-	if e.RequireEndpointRoute() {
+	if e.RequireEndpointRoute() || !e.RequireRouting() {
```

Comment:
This does not reflect what we have discussed.

Reply:
Since we require either end-point routing or enable routing we have done the changes like this, pls do suggest.

Reply:
Apologies, this one was good I think. I read too fast and didn't notice you had added the negation.